.github/workflows/bench.yml

```diff
@@ -58,6 +58,11 @@ jobs:
       - name: Copy benchmarks from new to old
         run: rm -rf ./old/tests/perf ; cp -r ./new/tests/perf ./old/tests/
 
+      - run: |
```

Comment:
This should not be part of the PR, right?

Reply:
Oh, yes. Removed. Thanks!